### PR TITLE
addressing additional pull request feedback.

### PR DESCRIPTION
**Pull request by ns-rse**
Incorporated feedback from @Robadob.

Not sure on etiquette/protocol for changing already published blog posts but if this is ok then please do approve, if not I'll retract the PR.

### CODE REVIEW

File: _posts/2022-04-19-linting.md [hunk 1]
```diff
@@ -147,17 +146,20 @@ save_random_numbers.py:3:0: C0411: standard import "import csv" should be placed
 Your code has been rated at 0.00/10
 ```
 
-The output tells us which module has been inspected on the first line and at the start of each subsequent line this is
-repeated, followed by the line the problem has been encountered, then the column followed by a somewhat cryptic error
-code and then a message about the problem before finishing with a more descriptive generic message associated with the
-error code.
+The output tells us which module has been inspected on the first line. Each subsequent line indicates
```
Comment (Robadob) [inline _posts/2022-04-19-linting.md:149]: It's not subsequent line?, all the parts are on the same line. token?

File: _posts/2022-04-19-linting.md [hunk 2]
```diff
@@ -271,7 +273,7 @@ If we add the following line it prevents PyLint from reporting the specific erro
 ```python
 import numpy as np
 
-# pylint: disable=unsubscriptable-object
+## pylint: disable=unsubscriptable-object
```
Comment (Robadob) [inline _posts/2022-04-19-linting.md]: find replace mistake?

File: _posts/2022-04-19-linting.md [hunk 3]
```diff
@@ -294,10 +296,10 @@ do so add the following...
 
 ```bash
 [MESSAGES CONTROL]
-# Disable the message, report, category or checker with the given id(s). You
-# can either give multiple identifiers separated by comma (,) or put this
-# option multiple times (only on the command line, not in the configuration
-# file where it should appear only once).
+## Disable the message, report, category or checker with the given id(s). You
```
Comment (Robadob) [inline _posts/2022-04-19-linting.md]: find replace mistake?

**Review by Robadob — APPROVED** [review]
Few small comments.

> Not sure on etiquette/protocol for changing already published blog posts but if this is ok then please do approve, if not I'll retract the PR.

It's really upto you, I don't think anyone has an issue with minor edits. If there were a more significant edit, you may want to add an edit notice but I don't think the content has significantly changed here. 

Likewise you can ignore any feedback you feel wrong, they're generally suggestions rather than mandatory requirements when it comes to blog/newsletter content.

File: _posts/2022-04-19-linting.md (hunk 1 above)
Comment (ns-rse) [reply _posts/2022-04-19-linting.md:149]:
```
 ❱ pylint save_random_numbers.py
************* Module save_random_numbers   <<< This is the first line I refer to
save_random_numbers.py:1:0: C0114: Missing <<< These are the subsequent lines I go on to explain
```

File: _posts/2022-04-19-linting.md (hunk 2 above)
Comment (ns-rse) [reply _posts/2022-04-19-linting.md]: > find replace mistake?

Very astute, thanks for noticing those.

**Pull request merged**